language/stdlib/modules/multi_account.move

```diff
@@ -0,0 +1,104 @@
+// Errors:
+// 0 -> ADDRESS_NOT_ALLOWED
+// 1 -> CURRENCY_NOT_HELD
+address 0x0:
+module MultiCurrencyHolding {
```

Comment:
This is going to be a key abstraction for VASPs, so I think it's worth brainstorming a bit to see if we can come up with a really descriptive/intuitive name. Couple of thoughts:
- `MultiCurrencyHolding` evokes key management for me because of "holding" part, when think this module is really about facilitating deposits/transfers.
- `PaymentRouter` is one alternative name that comes to mind. This module lets you route both incoming and outgoing payments through the address that contains the relevant resource. Although the module is implemented in nicely generic way that lets you manage multiple currencies, "routing" seems like the essence of the utility here. As a justification: this module would still be quite useful even if you are a VASP that only deals in one currency.

Reply:
I'm actually quite a fan of the `PaymentRouter` suggestion, since I agree that this is really what it's doing. I'll change it to that right now, and if we come up with a better name later we can rename.